src/Components/InputModel.re

```diff
@@ -1,6 +1,34 @@
 open Oni_Core;
 open Utility;
 
+let wordSeparators = " ./\\()\"'-:,.;<>~!@#$%^&*|+=[]{}`~?";
+
+let separatorOnIndex = (index, text) => {
+  String.contains(wordSeparators, text.[index]);
+};
```

Comment:
This can raise an exception since it indexes into the string without doing any bounds-checking. Not a big problem since it's just used internally, but ti should still be named to indicate so. The convention is to add an `Exn` suffix:

```suggestion
let separatorOnIndexExn = (index, text) => {
  String.contains(wordSeparators, text.[index]);
};
```

Reply:
Done